### PR TITLE
Fix packet blaster

### PR DESCRIPTION
**Pull request by ripatel-fd**

### CODE REVIEW

**Review by esemeniuc — APPROVED** [review]

**Review by segfaultdoc — APPROVED** [review]

**Review by jedleggett — APPROVED** [review]
tested

File: Cargo.toml [hunk 1]
```diff
@@ -49,10 +49,10 @@ openssl = "0.10.51"
 prost = "0.12.1"
 prost-types = "0.12.1"
 protobuf-src = "1.1.0"
-quinn = "0.9"
-rand = "0.8.5"
+quinn = "0.10"
+rand = "0.7"
```
Comment (jedleggett) [inline Cargo.toml:53]: why does this need downgrading?

Comment (jedleggett) [inline Cargo.toml:53]:
```suggestion
rand = "0.8.5"
```

Comment (esemeniuc) [reply Cargo.toml]: i think the rand change can be reverted unless solana libs need it

Comment (esemeniuc) [reply Cargo.toml]: https://github.com/search?q=repo%3Aquinn-rs%2Fquinn+%22rand+%3D%22&type=code

**Pull request closed without merging**